data/eos-autoupdater.conf

```diff
@@ -10,4 +10,5 @@
 #  3 - apply updates
 LastAutomaticStep=3
 IntervalDays=14
+RandomizedDelayDays=7
```

Comment:
Instead of changing the default behavior, let's only add the delay in the deployments where we need it (i.e. those that we turn on LAN configuration for)